tests/test_crawler.py

```diff
@@ -42,3 +43,19 @@ class CustomSettingsSpider(DefaultSpider):
 
         self.assertFalse(settings.frozen)
         self.assertTrue(crawler.settings.frozen)
+
+
+def SpiderManagerWithWrongInterface(object):
```

Comment:
Oh god, why did we merged this (should be 'class', got a little too functional there). I'm not even sure how the test is working.